_glossary/CHAKRA.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Chakra
+excerpt: JavaScript engine developed by Microsoft for it's Edge browser. Can optionally be used with Node.js instead of V8.
```

Comment:
I usually try to keep a single sentence for the description. Could we do:

> a JavaScript engine developed by Microsoft for its Edge browser which could also be used with Node.js instead of V8.